### ISSUE TITLE
task runner shows nothing

### ISSUE THREAD

**Issue by nigel-dewar** [JonCubed/angular2-cli-quick-start-aspnet-core-vs2015#1]
Hi There,

I cloned your repo. Installed all dependencies. Open task runner and find nothing.

does one need to go through manual setup with your instructions, or can one clone repo, install dependencies and easily get going?

Cheers

**Comment by JonCubed** [comment]
Yes it use to just work after clone and restoring dependencies, but this repo is very dated now. Angular 2, Angular CLI, ASP.NET Core have all had breaking changes since I did this. If I get some time in the next week I'll update it to the latest of all the frameworks.

**Comment by aoakeson** [comment]
Any update on this? Im trying to use the angular CLI in Core 1.0

**Comment by theDiverDK** [comment]
Any news about the ' If I get some time in the next week ' ?

Could really use your insights
